operators/operator-reference.adoc

```diff
@@ -31,6 +31,7 @@ include::modules/ingress-operator.adoc[leveloffset=+1]
 include::modules/kube-apiserver-operator.adoc[leveloffset=+1]
 include::modules/kube-controller-manager-operator.adoc[leveloffset=+1]
 include::modules/cluster-kube-scheduler-operator.adoc[leveloffset=+1]
+include::modules/cluster-cloud-controller-manager-operator.adoc[leveloffset=+1]
```

Comment:
Should this move up to line 19 to be alphabetical? Seems the list mostly follows that convention.

Reply:
Oh yeah I see it's mostly alpha, but the random O in the middle of the Cs threw me off. @adellape is the intent here to be alphabetical in general?